### ISSUE TITLE
data_summary: Exception: Internal Error 

### ISSUE THREAD

**Issue by truongc2** [data-describe/data-describe#440]
Your bug may already be reported!
Please search on the [issue tracker](../issues?q=label%3Abug+is%3Aissue) before creating one.

**Description**

When working with a modin dataframe object, data_summary() returns:
"Exception: Internal Error. Please email bug_reports@modin.org with the traceback and command that caused this error."

However, dd.data_summary(df, compute_backend="modin") works fine when df is a pandas dataframe and compute_backend="modin"

**Steps to Reproduce**

```python
import modin.pandas as mpd
from sklearn.datasets import load_boston
import data_describe as dd

data = load_boston()
df = mpd.DataFrame(data.data, columns=list(data.feature_names))
dd.data_summary(df)
```
![image](https://user-images.githubusercontent.com/38387928/102284823-893aa480-3efa-11eb-9e18-62b56b57d3d7.png)

**Your Environment**
* data describe version: 0.1.0b3
* Python version: 3.7.7
* Operating System: MacOS

**Comment by haishiro** [comment]
@truongc2 Is this related to #389? You could test by running with modin >=0.8.2 which was just released a month ago.

**Comment by truongc2** [comment]
> @truongc2 Is this related to #389? You could test by running with modin >=0.8.2 which was just released a month ago.

You're right. It's related to #389. I'll integrate the update into this PR
